features/bootstrap/FeatureContext.php

```diff
@@ -355,6 +421,66 @@ public function thereIsDummyObjectsWithDummyBoolean($nb, $bool)
     }
 
     /**
+     * @Given there is :nb embedded dummy objects with embeddedDummy.dummyBoolean :bool
+     */
+    public function thereIsDummyObjectsWithEmbeddedDummyBoolean($nb, $bool)
+    {
+        if (in_array($bool, ['true', '1', 1], true)) {
```

Comment:
Okay so did some digging. This actually comes from https://github.com/api-platform/core/pull/1010 and I used the same code as we already use in the `BooleanFilter` for consistency.